grocy-chores-card.js

```diff
@@ -103,7 +103,7 @@ customElements.whenDefined('card-tools').then(() => {
                     : ""}
                   </div>
                   <div>
-                    <mwc-button @click=${ev => this._track(chore.id)}>${this.translate("Track")}</mwc-button>
+                    <mwc-button @click=${ev => this._track(chore.id, chore.next_execution_assigned_user.id)}>${this.translate("Track")}</mwc-button>
```

Comment:
I'm afraid that this will throw an exception if next_execution_assigned_user is null (which it can be), so we should guard against that.